mobile/assets/locales/ro/translations.json

```diff
@@ -163,7 +163,8 @@
         "error": {
           "arrival_first": "Te rugăm să selectezi mai întâi ora sosirii.",
           "later_departure": "Te rugăm să selectezi o oră de plecare ulterioară orei de sosire.",
-          "earlier_arrival": "Te rugăm să selectezi o oră de sosire anterioară orei de plecare."
+          "earlier_arrival": "Te rugăm să selectezi o oră de sosire anterioară orei de plecare.",
+          "server": "Nu am putut trimite informatiile legate de ora sossirii si ora plecarii."
```

Comment:
Trebuie adaugata cheia de translate pe toate limbile si cerut translation direct in PR pana sa-i facem merge. 

Typo la "sossirii"